frontend/app/rollup.config.js

```diff
@@ -359,26 +348,33 @@ export default {
 
         production && filesize(),
 
-        // If we're building for production, copy sourcemaps to '_/raw'
-        // and update the js files to point to the new sourcemap locations
-        production &&
-            copy({
-                targets: [
-                    {
-                        src: "build/*.map",
-                        dest: "build/_/raw",
-                    },
-                    {
-                        src: "build/*.js",
-                        dest: "build",
-                        transform: (contents, filename) =>
-                            contents
-                                .toString()
-                                .replace("//# sourceMappingURL=", "//# sourceMappingURL=./_/raw/"),
-                    },
-                ],
-                hook: "writeBundle",
-            }),
+        // Pull in the worker and push_sw
+        copy({
```

Comment:
Now that I see this I have a nasty feeling that I tried this and it didn't work in some way (maybe in dev mode or something) and that's why I flipped it round the other direction. Maybe it won't be a problem any more.

Reply:
Seems fine to me locally and on prod test.